lib/matplotlib/ticker.py

```diff
@@ -1016,33 +1039,29 @@ def __call__(self, x, pos=None):
         else:
             base = '%s' % b
 
-        if coeff in self.sublabel:
-            if not is_decade and self.labelOnlyBase:
```

Comment:
It's been fixed. I renamed `is_decade` *and* `isDecade` into `is_x_decade`
In general I find the variable names of this module very hard to understand. I'll plan on doing some clean up after 2.0